scss/initial/_root.scss

```diff
@@ -30,6 +31,7 @@
 /// @group style-typography
 :root {
   @include tools.font-family('sans');
+  @include tools.colors--(colors.$demo-colors);
```

Comment:
This maybe doesn't belong here? Not sure.

Reply:
@dvdherron I think the intent is to _not_ load them inside the iframe styles. The distinction is that anything included in `iframes.scss` will be made available within the iframe for _every project that uses Herman_. Anything that is specific to Herman's own documentation of itself (easily confused with Herman's role documenting other projects) should be in `main.scss`.

Reply:
Moved to custom-props specific stylesheet in https://github.com/oddbird/sassdoc-theme-herman/commit/d5f6a2b4104c078414f2603781d0ce38c7467193